tf-modules/archive/ems_reporting.tf

```diff
@@ -77,6 +77,8 @@ resource "aws_lambda_function" "ems_product_metadata_report" {
       CollectionsTable    = var.dynamo_tables.collections.name
       cmr_client_id       = var.cmr_client_id
       cmr_provider        = var.cmr_provider
+      cmr_username        = var.cmr_username
+      cmr_password        = var.cmr_password
```

Comment:
I don't see where these environment variables are used in the Lambda?

Reply:
You're right. These use the CMRSearchConceptQueue, which I have not updated yet, will do in next PR. I removed this for now since it has to use the secrets password anyway as Matt had pointed out.